include/boost/wintls/certificate.hpp

```diff
@@ -11,9 +11,18 @@
 #include <boost/wintls/error.hpp>
 #include <boost/wintls/file_format.hpp>
 
+
+#include <boost/wintls/detail/config.hpp>
 #include <boost/wintls/detail/win32_crypto.hpp>
 
 #include <memory>
+#include <cassert>
+
+#if defined(NDEBUG)
+#define WINTLS_VERIFY_MSG(expr, msg) ((void)(expr))
+#else
+#define WINTLS_VERIFY_MSG(expr, msg) assert((expr) && (msg))
```

Comment:
This should be moved to a different header. Not sure if one of the existing ones would make sense or a new one should be added though.